wundergraph_cli/Cargo.toml

```diff
@@ -17,7 +17,7 @@ diesel = "1.4"
 
 [dev-dependencies]
 dotenv = "0.15"
-insta = "0.12"
+insta = "0.16"
```

Comment:
I'm afraid this commit has an impact on snapshot names: your snapshot was named [tests__infer_schema.snap](https://github.com/weiznich/wundergraph/blob/master/wundergraph_cli/src/print_schema/snapshots/tests__infer_schema.snap), but insta v0.16 seems to look for `wundergraph_cli__print_schema__tests__infer_schema.snap`

Reply:
That's totally fine. Just move/regenerate the snapshot files to the new location.